rotkehlchen/globaldb/handler.py

```diff
@@ -2103,3 +2109,13 @@ def delete_location_asset_mappings(entries: list[LocationAssetMappingDeleteEntry
                         f'Failed to delete the location asset mapping of {entry.location_symbol} in '  # noqa: E501
                         f'{entry.location} because it does not exist in the DB.',
                     )
+
+    @staticmethod
+    def is_asset_symbol_unsupported(location: 'Location', asset_symbol: str) -> bool:
+        """Returns True if the asset with the given symbol is not supported in the given location.
+        Otherwise returns False."""
```

Comment:
Keep it simple

```suggestion
        """Returns if the asset with the given symbol is not supported in the given location"""
```

Reply:
Resolved here https://github.com/rotki/rotki/pull/7628